src/components/Blog/Blog.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Link } from 'gatsby'
+
+import './Blog.css'
+
+export function Blog() {
+  return (
+    <article className="blog">
+      <h2>blog</h2>
+      <div className="blog__content">
+        <p>
+          Find out about open source with our <Link to="/blog">blog</Link>.
```

Comment:
**Change text about:**
- Why we have written this white paper ?
- Add links to articles with all available chapters (chapter number, title, summary)